src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -401,11 +401,11 @@ internal void Save(ModelSaveContext ctx)
                 var active = new bool[1];
                 string[][] friendlyNames = _columns.Select(c => c.FriendlyNames).ToArray();
                 // We will create invert hash helper class, which would store in itself all original ngrams and their mapping into hash values.
-                var helper = new InvertHashHelper(this, input.Schema, friendlyNames, sourceColumnsForInvertHash.Contains, invertHashMaxCounts);
+                var helper = new InvertHashHelper(this, input.Schema, friendlyNames, c => sourceColumnsForInvertHash.Any(col => col.Index == c), invertHashMaxCounts);
```

Comment:
>c => sourceColumnsForInvertHash.Any(col => col.Index == c) [](start = 85, length = 58)

or you can just modify InvertHashHelper and accept list of columns instead of predicate. It's get used once.